ale_linters/go/golint.vim

```diff
@@ -0,0 +1,45 @@
+" Author: neersighted <bjorn@neersighted.com>
+" Description: golint for Go files
+
+if exists('g:loaded_ale_linters_go_golint')
+    finish
+endif
+
+let g:loaded_ale_linters_go_golint = 1
+
+function! ale_linters#go#golint#Handle(buffer, lines)
```

Comment:
The issue with that is one of severity. I guess I can add two wrapper functions.